distro/depends/kernel-selftests

```diff
@@ -4,20 +4,20 @@ gcc-multilib
 libc6-dev
 libc-bin
 libc6-dev-i386 (x86_64)
-libc6-dev-x32
```

Comment:
thanks, why does this depends/kernel-selftests need to be changed in this patch? Like is it part of lkp install?

Reply:
no it's not, just i saw this has the same issue as `depends/lkp-dev` by a quick grep: these packages are only available in amd64 arch repositories but not marked with (x86_64) as the code logic and some other package (libc6-dev-i386). So I made the changes together.

I've checked these packages on amd64 and riscv port of Ubuntu, riscv port doesn't have these packages though some similar name packages exist. Debian should be the same. But other distros like centos/fedora are not changed because I don't use them.

Reply:
got it, prefer to have a separate patch for this file, when solving the failure from lkp install kernel-selftests.yaml